discovery-provider/src/queries/get_saves.py

```diff
@@ -50,6 +50,7 @@ def get_saves(save_type, user_id):
                     session.query(Track.track_id).filter(Track.is_current == True)
                 )
             )
+        query = query.order_by(Save.created_at.desc())
```

Comment:
given that this is paginated, is it a problem that they're going to come back in a different order now?

We also don't have an index on the created_at field, so I would be a little worried about performance here for users with a big bunch of saves. We could do a migration to add the index if this is necessary?

Reply:
Yeah I had this concern as well. Index sounds like the right answer

Reply:
blocknumber is indexed tho so we can get away with using that for sorting

Reply:
Would the most recent favorite be the latest block?

Reply:
yup created_at / blocknumber are essentially the same

https://github.com/AudiusProject/audius-protocol/blob/main/discovery-provider/src/tasks/entity_manager/social_features.py#L74-L75

oh and also if you want this to be deterministic, it should probably be sorted by both created_at and track_id. that way saves that were indexed in the same block have a deterministic order.

Reply:
incredible. @isaacsolo to the rescue 🙏